napari/_qt/_qapp_model/qactions/__init__.py

```diff
@@ -51,5 +52,5 @@ def _provide_qt_viewer() -> Optional[QtViewer]:
         return None
 
     # register actions
-    for action in chain(Q_VIEW_ACTIONS, Q_HELP_ACTIONS):
+    for action in chain(Q_FILE_ACTIONS, Q_HELP_ACTIONS, Q_VIEW_ACTIONS):
```

Comment:
Maybe a silly question but why can we not simply do:
```
app.register_actions(chain(Q_FILE_ACTIONS, Q_HELP_ACTIONS, ...))
```
below? Since `register_actions` seems just to be the 'plural' version of `register_action` ??

@DragaDoncila

Reply:
lgtm - can't we? Definitely try it...

Reply:
Done, let's see if anything breaks :woman_shrugging: